BRLS/src/library/Main.java

```diff
@@ -4,9 +4,9 @@
 import library.borrowitem.bORROW_IteM_cONTROL;
 import library.entities.*;
 import library.fixitem.FixItemUI;
-import library.fixitem.fIX_iTeM_cONTROL;
+import library.fixitem.FixItemControl;

```

Comment:
As per the class diagram, this the the correct class name.

Reply:
i can see it